engine/resources/exec.go

```diff
@@ -138,6 +142,10 @@ func (obj *ExecRes) Validate() error {
 		}
 	}
 
+	// check that environment variables' format is valid
+	for key := range obj.Env {
+		isNameValid(key)
```

Comment:
Missing something here...

Reply:
return, right?

Reply:
errwrap.Wrapf...

Reply:
Is it not redundant to have error messages in the `isNameValid` function _and_ in the errwrap?

Reply:
this currently does nothing

Reply:
I changed it in the most recent pull request to 
```
for key := range obj.Env {
	return errwrap.Wrapf(isNameValid(key), "invalid variable name")
}
return nil
```